packages/theme/src/baseTheme.ts

```diff
@@ -24,6 +24,7 @@ import {
 } from './types';
 import { pageTheme as defaultPageThemes } from './pageTheme';
 
+const DEFAULT_HTML_FONT_SIZE = 16;
```

Comment:
is this actually in active use (by ourselves)? 🤔 is it here just for completeness?

Reply:
Resolved here: https://github.com/backstage/backstage/pull/11510#discussion_r893472816.